rocky/reports/templates/partials/report_severity_totals_table.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+
+<div class="horizontal-scroll">
+    <table>
+        <caption class="visually-hidden">{% translate "Total per severity overview:" %}</caption>
+        <thead>
+            <tr>
+                <th>{% translate "Severity" %}</th>
+                <th>{% translate "Unique" %}</th>
+                <th>{% translate "Total occurrences" %}</th>
```

Comment:
Not sure what we use in other tables, but perhaps because I was quickly wondering what "Unique" means we could instead say "Unique/Total Finding Types" or perhaps "Finding Types" even?

Reply:
If that makes the table (and the code) more clear, I'll change it (also in the report.py). 

For continuity, I'd then go for "Finding types" and "Occurrences" (without the "Total"). What do you think?

Reply:
I think that would indeed make sense!